### PR TITLE
[kube-prometheus-stack] add support for configure services externalTr…

### PR DESCRIPTION
**Pull request by alonbhaim**
…afficPolicy using values

Signed-off-by: Alon Ben Haim <alon.benhaim@kaltura.com>

#### What this PR does / why we need it:
Sometimes when using services we need the ability to control the external traffic policy of them, I added the following option using values.yaml for all relevant services in the chart.

#### Which issue this PR fixes
* will close that #https://github.com/prometheus-community/helm-charts/issues/1755 issue when PR gets merged 

#### Checklist
- [X] [DCO] signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name

### CODE REVIEW

**Comment by yurrriq** [comment]
@alonbhaim, can you bump the chart version? We need this too.

**Comment by alonbhaim** [comment]
> @alonbhaim, can you bump the chart version? We need this too.

Sure, I'll pull changes from main branch to my fork and will push new version.

**Comment by alonbhaim** [comment]
@yurrriq merged lastest changes from upstream and bumped the version.

**Comment by yurrriq** [comment]
Thanks! Now we just need a code owner to approve.

**Comment by alonbhaim** [comment]
@monotek can you re-run the lint checks ?

**Review by monotek — APPROVED** [review]

**Comment by alonbhaim** [comment]
Hi,
Can you please review ? @andrewgkew @bismarck @desaintmartin @desaintmartin @gianrubio @gkarthiks @scottrigby @Xtigyro

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -13,8 +13,6 @@ maintainers:
     email: gianrubio@gmail.com
   - name: gkarthiks
     email: github.gkarthiks@gmail.com
-  - name: GMartinez-Sisti
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:16]: please add back

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — APPROVED** [review]

**Pull request merged**